playbooks/Makefile

```diff
@@ -0,0 +1,25 @@
+.PHONY = monitoring
+
+.DEFAULT_GOAL = help
+
+## Install monitoring on customer cluster. Example: CUSTOMER=qa ENVIRONMENT=production TIER=basic PLATFORM=azure make monitoring
+monitoring:
+	@@if test -z "$$CUSTOMER"; then echo "ERROR: CUSTOMER is not defined."; exit 1; fi;
+	@@if test -z "$$PLATFORM"; then echo "ERROR: PLATFORM is not defined."; exit 1; fi;
+	@@if test -z "$$TIER"; then echo "ERROR: TIER is not defined."; exit 1; fi;
+	@@if test -z "$$ENVIRONMENT"; then echo "ERROR: ENVIRONMENT is not defined."; exit 1; fi;
+	@ansible-playbook -i inventory monitoring.yml --extra-vars "@../../edx-configs/$$CUSTOMER/$$ENVIRONMENT/server-vars.yml"
```

Comment:
Caveat: Some customers in edx-configs are structured as `$CUSTOMER/$ENVIRONMENT/files/server-vars.yml`. We'll have to address this when refactoring edx-configs.